src/liberty/devCommands.ts

```diff
@@ -130,7 +129,7 @@ export async function startDevMode(libProject?: LibertyProject | undefined): Pro
 
 export async function removeProject(): Promise<void> {
     const projectProvider: ProjectProvider = ProjectProvider.getInstance();
-    
+
```

Comment:
Please do not do any formatting changes to code that is not added or modified by you for this PR.